app/src/main/java/com/example/myapplication/MatchData.java

```diff
@@ -142,24 +146,64 @@ public void setDump(int dump) {
     }
 
     public NoteAcquisition getNoteAcquired() {   return noteAcquired;    }
-    public void setNoteAcquired(NoteAcquisition noteAcquired) {   this.noteAcquired = noteAcquired;   }
+    public void setNoteAcquired(boolean floor, boolean source) {
+        if (floor && source){
+            this.noteAcquired =  NoteAcquisition.Both;
+        } else if (floor && !source) {
+            this.noteAcquired =  NoteAcquisition.Floor;
+        } else if (!floor && source) {
+            this.noteAcquired =  NoteAcquisition.Source;
+        } else {
+            this.noteAcquired = NoteAcquisition.None;
+        }
+    }
 
     //Endgame
 
     public Chain getChaining() {    return chaining;    }
-    public void setChaining(Chain chaining) {    this.chaining = chaining;    }
+    public void setChaining(int index) {
```

Comment:
Why changing from the enum value to the integer value? It will be hard to use this method